playbooks/common/openshift-master/config.yml

```diff
@@ -69,6 +73,11 @@
       openshift_env:
         openshift_hosted_registry_storage_kind: 'nfs'
     when: openshift_hosted_registry_storage_kind is not defined and groups.oo_nfs_to_config is defined and groups.oo_nfs_to_config | length > 0
```

Comment:
@detiber Yep and this future group leak will use a var representing the list of nfs hosts | default([]).